### PR TITLE
add change log for coming 3.17.1 release

### PR DESCRIPTION
**Pull request by drelaptop**

### CODE REVIEW

**Comment by drelaptop** [comment]
change log is important to show a release changes for developers, could you read this to check something leak, or wrong expression? @PatriceJiang @slackmoehrle

**Review by PatriceJiang — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,76 @@
...
+[REFINE]        Android: add supporting SafeArea for most edge screens devices
+[REFINE]        Android: change default value of PROP_BUILD_TYPE from ndk-build to CMake
+[REFINE]        Android: update project config to gradle 4.6, sdk 28, android gradle plugin 3.1.0
+[REFINE]        Android: add ability to not stop music on game start in android
+[REFINE]        Windows: refine CCDevice-win32 BitmapDC
+[REFINE]        Windows: improve binary folder structure, copy resources to "Resources/" folder
+[REFINE]        Action: add ui::LoadingBar support to Progress actions
+[REFINE]        Network: support multiple parameters in SocketIO emit/send method
+[REFINE]        FileUtils: improve thread safety (use single recursive_mutex)
+[REFINE]        FileUtils: add listFiles impl for Android and Windows platform
```
Comment (PatriceJiang) [inline CHANGELOG:23]: FileUtils  add `getFileSize` impl for Windows

Comment (drelaptop) [reply CHANGELOG:23]: thanks, I will add this item later.

**Comment by drelaptop** [comment]
only changlog modified, not any code changes, so merge it directly.

**Comment by summerinsects** [comment]
hey, November does not have date 31

**Comment by drelaptop** [comment]
thanks for review, that's a stupid mistake, I will correct it.

**Pull request merged**